automation/cleanDockerfile/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fedora:latest
+MAINTAINER Shaked Lokits "shaked.lokits@gmail.com"
+
+
+###############################################################################
+############################ Initial Installations ############################
+###############################################################################
+
+RUN dnf -y update
+RUN dnf -y install openssh-server git java-1.8.0-openjdk.x86_64 which
```

Comment:
.x86_64 ?
which?

Reply:
I really don't know why Shaked wanted to install 'which' command, In fact, It is not disturbing me

Reply:
I assume that he used it in the Jenkinsfile or in JJB files

Reply:
or debugging. remove if not required